server/src/test/java/org/elasticsearch/search/SearchServiceTests.java

```diff
@@ -56,6 +56,7 @@
 import org.elasticsearch.search.aggregations.support.ValueType;
 import org.elasticsearch.search.builder.SearchSourceBuilder;
 import org.elasticsearch.search.fetch.ShardFetchRequest;
+import org.elasticsearch.search.fetch.subphase.ScriptFieldsContext;
```

Comment:
Looks like the checkstyle runner doesn't like unused imports, could you remove this line please?

```
17:52:18 [ant:checkstyle] [ERROR] /var/lib/jenkins/workspace/elastic+elasticsearch+pull-request/server/src/test/java/org/elasticsearch/search/SearchServiceTests.java:59:8: Unused import - org.elasticsearch.search.fetch.subphase.ScriptFieldsContext. [UnusedImports]
```

Reply:
My bad. Done!